README.md

```diff
@@ -465,6 +465,7 @@ Here is a table of the [diagnostic data we collect][diagnosticdata]:
 | `is_ci`               | Whether the installer is being used in CI (e.g. GitHub Actions).                                      |
 | `action`              | Either `Install` or `Uninstall`.                                                                      |
 | `status`              | One of `Success`, `Failure`, `Pending`, or `Cancelled`.                                               |
+| `attribution`         | Optionally defined by the user, associate the diagnostics of this run to the provided value.          |
```

Comment:
https://github.com/DeterminateSystems/nix-installer/blob/1c103edb9033eb4f954e14e9de94535b3f06c462/src/settings.rs#L208-L234 needs to be updated too.

Reply:
Nice, done!